### PR TITLE
add new elements

### PR DESCRIPTION
**Pull request by aaronflorey**
Added Previously Shown, Episode Num, Premiere, Icon, New

Only problem is the XML doesn't pass validation if you add episode-num and i can't figure out why

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -13,4 +13,4 @@ trim_trailing_whitespace = true
 trim_trailing_whitespace = false
 
 [*.xml]
-indent_size = 2
+indent_size = 1
```
Comment (jaylinski) [inline .editorconfig:16]: Why did you change this?

File: src/Tv/Programme/PreviouslyShown.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+
```
Comment (jaylinski) [inline src/Tv/Programme/PreviouslyShown.php:3]: One line too much. (See style-ci error)

File: src/Tv/Programme/PreviouslyShown.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+namespace XmlTv\Tv\Programme;
+
+
```
Comment (jaylinski) [inline src/Tv/Programme/PreviouslyShown.php:6]: One line too much. (See style-ci error)

File: spec/xmltv.dtd [hunk 4]
```diff
@@ -74,15 +74,15 @@
...
+               height      CDATA #IMPLIED>
 
 <!ELEMENT value (#PCDATA)>
 
 <!ELEMENT country (#PCDATA)>
 <!ATTLIST country lang CDATA #IMPLIED>
 
 <!ELEMENT episode-num (#PCDATA)>
-<!ATTLIST episode-num system CDATA "onscreen">
+<!ATTLIST episode-num system CDATA #REQUIRED>
```
Comment (jaylinski) [inline spec/xmltv.dtd:85]: Where did you get this change? Or was it only a test because of your validation issues?

Original: https://github.com/XMLTV/xmltv/blob/master/xmltv.dtd

**Review by jaylinski — COMMENTED** [review]
Thanks for your contribution. Please answer my questions in the review.
After that I'll try to help you debug the validation issue.

**Comment by aaronflorey** [comment]
Sorry for the late reply.

The styling errors and extra changes were my IDE, I'll fix them up.

File: spec/xmltv.dtd (hunk 4 above)
Comment (aaronflorey) [reply spec/xmltv.dtd:85]: Yeah i was testing here. I've never worked with DTD before, if you have it as "onscreen" does that mean it defaults to onscreen, or the attribute has to equal onscreen?

Comment (jaylinski) [reply spec/xmltv.dtd:85]: Yes, as far as I can tell, it's a default value:
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms256140(v%3Dvs.100)

**Comment by jaylinski** [comment]
@Mochaka I published a new version which contains all missing elements: https://github.com/jaylinski/xmltv/releases/tag/v2.0.0-alpha.1

If there are problems, please open a new issue.

**Comment by jaylinski** [comment]
@Mochaka I published a new stable version: https://github.com/jaylinski/xmltv/releases/tag/v2.0.0
